src/lib.rs

```diff
@@ -117,6 +117,11 @@ pub mod hash_set {
 pub use crate::map::HashMap;
 pub use crate::set::HashSet;
 
+#[cfg(feature = "equivalent")]
+use equivalent::Equivalent;
```

Comment:
Shouldn't this be `pub` to match the API without the feature?
(or else hide them both in a breaking change)